src/app/services/cinchy-query.service.ts

```diff
@@ -105,6 +125,6 @@ export class CinchyQueryService {
     const params = {
       '@formId': sessionStorage.getItem('formId')
     };
-    return this.cincyService.executeQuery('Cinchy Forms', 'Get Form Sections', params);
+    return this.cincyService.executeQuery('Cinchy Forms', 'Get Form SectionsNew', params);
```

Comment:
keep "Get Form Sections", you don't want the actual code to point to "Get Form SectionsNew"